modules/nw-udn-support-matrix-primary-secondary.adoc

```diff
@@ -0,0 +1,324 @@
+//module included in the following assembly:
+//
+// *networkking/multiple_networks/understanding-user-defined-networks.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="support-matrix-for-primary-secondary-networks_{context}"]
+= Support matrix for primary and secondary networks
+
+The `UserDefinedNetwork` and `NetworkAttachmentDefinition` custom resources (CRs) provide cluster administrators and users the ability to create customizable network configurations and define their own network topologies, ensure network isolation, manage IP addressing for workloads, and leverage advanced network features. A third CR, `ClusterUserDefinedNetwork`, is also available, which allows administrators the ability to create and define additional networks spanning multiple namespaces at the cluster level. Both user-defined networks and network attachment definitions can serve as both the primary and secondary network interface, and each support `layer2` and `layer3` topologies; a third network topology, Localnet, is also supported with network attachment definitions. 
+
+[NOTE]
+====
+As of {product-title} 4.18, the Localnet topology is unavailable for use with the `UserDefinedNetwork` and `ClusterUserDefinedNetwork` APIs. It is available for use with the `NetworkAttachmentDefinition` API only. 
+====
+
+The following section highlights the supported features of the `UserDefinedNetwork` and `NetworkAttachmentDefinition` CRs when they are used as either the primary or secondary network. A separate table for the `ClusterUserDefinedNetwork` CR is also included. 
+ 
+.Primary network support matrix
+[cols="1,1a,1a,1a,1a,1a, options="header"]
+|===
+| Network type | Network feature | Supported | Layer2 topology |Layer3 topology |Localnet topology ^[1]^
+
+
+.10+| Primary network (UDN and NAD)
```

Comment:
you cannot have localnet as primary network, not even with NAD.

Reply:
I have put an "X" for all localnet features. I might remove it from the table entirely, then.